### PR TITLE
Lint Error fixes

### PR DESCRIPTION
**Pull request by peRFectBeliever**
Lint error fixes

### CODE REVIEW

File: virtualbox/__init__.py [hunk 1]
```diff
@@ -24,16 +24,16 @@
...
-    __title__,  # noqa: F401
-    __version__,
-    __author__,
-    __author_email__,
-    __maintainer__,
-    __maintainer_email__,
-    __license__,
-    __url__,
-)
+# from .__about__ import (
```
Comment (sethmlarson) [inline virtualbox/__init__.py:27]: Commenting this out isn't a lint error fix?

File: virtualbox/library_base.py [hunk 2]
```diff
@@ -26,10 +26,10 @@ class EnumType(type):
 
     def __init__(cls, name, bases, dct):
         cls._value = None
-        cls._lookup_label = dict((v, l) for l, v, _ in cls._enums)
-        cls._lookup_doc = dict((v, d) for _, v, d in cls._enums)
-        for l, v, _ in cls._enums:
-            setattr(cls, pythonic_name(l), cls(v))
+        cls._lookup_label = dict((vl, lu) for lu, vl, _ in cls._enums)
```
Comment (sethmlarson) [inline virtualbox/library_base.py:29]: Let's switch to proper names like `value` and `label`?

**Review by sethmlarson — CHANGES_REQUESTED** [review]
Thanks, two comments.

File: virtualbox/__init__.py (hunk 1 above)
Comment (peRFectBeliever) [reply virtualbox/__init__.py]: I have unmommented and fixed  errors

File: virtualbox/library_base.py (hunk 2 above)
Comment (peRFectBeliever) [reply virtualbox/library_base.py]: completed

**Comment by peRFectBeliever** [comment]
completed review comments, kindly merge those changes

**Comment by peRFectBeliever** [comment]
Could you please merge this

**Pull request closed without merging**